examples/quickstart.php

```diff
@@ -34,14 +34,11 @@ class User extends AggregateRoot
 
         /**
          * ARs should be created via static factory methods
-         *
-         * @param string $username
-         * @return User
          */
-        public static function nameNew($username)
+        public static function nameNew(string $username): User
         {
             //Perform assertions before raising a event
-            \Assert\that($username)->notEmpty()->string();
+            Assertion::notEmpty($username);
```

Comment:
This line should be replaced, since the "beberlei/assert" dependency is not available in composer anymore.